### ISSUE TITLE
Cannot run AB or AB_DEV on MAC

### ISSUE THREAD

**Issue by nanocate** [LCA-ActivityBrowser/activity-browser#714]
Hello, 

I followed the steps to install and run the activity browser on my mac. I am unable to do so... See the error message below:

<img width="973" alt="Screenshot 2022-02-14 at 15 37 53" src="https://user-images.githubusercontent.com/7212378/154027772-ba33b8f6-4835-4f88-b98f-9c678a86b627.png">

<img width="1004" alt="Screenshot 2022-02-14 at 14 33 31" src="https://user-images.githubusercontent.com/7212378/154027920-61ff3cd1-5683-419e-9d57-ba0d11f1bafc.png">

Thanks for your help!

**Comment by haasad** [comment]
I suspect this is a problem with python 3.10. Could you try re-creating the environment with python 3.9?
```
conda deactivate
conda remove -n ab_dev --all
conda create -n ab_dev activity-browser-dev python=3.9
```

**Comment by nanocate** [comment]
Cool. Thanks for the tip! That was the issue

**Comment by haasad** [comment]
Glad it worked. We'll look for a permanent fix for this, the AB shouldn't create an environment with python 3.10 if it doesn't work with it (yet).

**Comment by cmutel** [comment]
@haasad This is fixed as of `bw2data` [3.6.4](https://github.com/brightway-lca/brightway2-data/commit/71be678f2540d95d2f61ade565a35a1f399d9f2a). We should be able to 3.10 now.

**Comment by haasad** [comment]
See my comment in https://github.com/LCA-ActivityBrowser/activity-browser/pull/722#issue-1142857142

python 3.10 doesn't work yet. windows can't resolve all packages and the tests for linux and osx don't finish
